purchase_quick/views/purchase_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="view_purchase_form" model="ir.ui.view">
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <field name="order_line" position="before">
+                <button
+                    name="add_product"
+                    string="Add"
```

Comment:
Suggested improvement : "Quick add" or "Add from supplier's product list" ?

Reply:
Indeed. better explicit that implicit. Changed. (I put "Add from supplier's product list")